.chainsaw.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: chainsaw.kyverno.io/v1alpha1
+kind: Configuration
+metadata:
+  creationTimestamp: null
+  name: configuration
+spec:
+  # TODO: why skip delete should be set to true ?
+  # skipDelete: true
```

Comment:
We do have two kuttl configuration one for local debugging keeps all the resources so that you can check them manually, the other for pipeline or testing cleans up everything, can it be that this is inherited form it? Also currently we export most cluster resources after the test runs in case of falure because we did find the collecture feature a bit limiting, however I see chainsaw has a better way to log the test so we could review this logic

Reply:
Thanks @RealAnna !
I will look at that in more depth soon, just wrote that down while looking at the converted file :)

Reply:
Fixed